t/mojo/stream_while_stopped.t

```diff
@@ -0,0 +1,12 @@
+use Mojo::Base -strict;
+
+use Test::More;
+use Mojo::IOLoop;
+
+Mojo::IOLoop->client({address => 'localhost'}, sub {});
```

Comment:
Is this actually portable?

Reply:
On second thought, the whole test seems completely pointless since it doesn't even verify that `Net::DNS::Native` is installed.

Reply:
To properly verify it works the same with and without you need two test files (along the lines of poll/ev tests), and find a scenario that doesn't rely on private instance data. And it would prolly have to be online tests, so the tests can be used to cover a few more interesting DNS cases.